src/settings-ui/Microsoft.PowerToys.Settings.UI/Helpers/StartProcessHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace Microsoft.PowerToys.Settings.UI.Helpers
+{
+    public static class StartProcessHelper
+    {
+        public const string ColorsSettings = "ms-settings:colors";
+
+        public static void Start(string process)
```

Comment:
We may use this class for opening other settings pages as well, let's rename it
```suggestion
        public static void OpenColorSettings(string process)
```